### PR TITLE
add owner and specie table

### PR DESCRIPTION
**Pull request by chumaachike**
Added the Owner and Specie table ✔✔✔
Removed the specie column in animal table ✔✔✔✔
Ran queries as stipulated, please check out images to see results

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,4 +11,22 @@
...
  INSERT INTO animals (id, name, date_of_birth, escape_attempts, nutered, weight_kg) VALUES (11, ' Ditto', '2005-07-01', 7, true, 20);
-
+/* Insert into owners table */
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 77);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 77);
+/* If the name ends in "mon" it will be Digimon */
```
Comment (Danieldotcomcoder) [inline data.sql:21]: Kindly insert 'Pokemon' and 'Digimon 'into the species table , as requested in the project requirements 👍 😄

File: queries.sql [hunk 2]
```diff
@@ -27,12 +27,16 @@ SELECT AVG(weight_kg) FROM animals;
...
+SELECT AVG(escape_attempts) FROM animals WHERE species = 'digimon' AND date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name='Melody Pond';
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name='Pokemon';  
+SELECT COUNT (*) FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT COUNT (*) FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Digimon';
+ SELECT animals.name from animals JOIN species ON animals.species_id = species.id JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+ SELECT animals.name from animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts<1;
+ SELECT owners.full_name FROM owners JOIN animals ON owners.id = animals.owner_id WHERE animals.escape_attempts = ( SELECT MAX(animals.escape_attempts) FROM animals);
```
Comment (Danieldotcomcoder) [inline queries.sql:42]: Kindly fix this query as it doesn't answer 'Who owns the most animals?' 👍 
You can add Count for this query . 😄

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @chumaachike , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**